sdk/mongocluster/arm-mongocluster/src/static-helpers/serialization/build-ssv-collection.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function buildSsvCollection(items: string[] | number[]): string {
```

Comment:
why we have this change? could you verify if these helpers are used in src?

Reply:
no using in src